src/server/graph-generation/metadataMapperJson.js

```diff
@@ -152,7 +152,7 @@ function getElementFromBioPax(biopaxFile, id) {
   }
 
   //Get element matching the id
-  let result = biopaxFile.filter(data => data['pathid'] === id);
+  let result = biopaxFile.filter(data => data['pathid'].indexOf(id) !== -1);
```

Comment:
no, data['pathid'] in the biopax file is a String.

Reply:
So this is checking if data['pathid'] contains the id?

If so it might make sense to replace this with ```string.includes()```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes

Reply:
Yes, its checking if the biopax id contains a given formatted id.

I've replaced it with includes.